dropgz/build/cni.Dockerfile

```diff
@@ -8,17 +8,23 @@ RUN CGO_ENABLED=0 go build -a -o bin/azure-ipam -trimpath -ldflags "-X main.vers
 
 FROM mcr.microsoft.com/oss/go/microsoft/golang:1.18 AS azure-vnet
 ARG VERSION
+ARG OS
+ARG ARCH
+ENV OS=${OS}
```

Comment:
do you need to reassign the arg to an env variable like this? is it because we use it in a RUN?

Reply:
oh, just ran make without the env variable assignment and was able to build without those lines. removed it from the pr